tests/e2e_logging.py

```diff
@@ -197,6 +200,80 @@ def test_anonymous_caller(network, args):
     return network
 
 
+@reqs.description("Use multiple auth types on the same endpoint")
+@reqs.supports_methods("multi_auth")
+def test_multi_auth(network, args):
+    if args.package == "liblogging":
```

Comment:
Minor: perhaps log a warning if it's not the case.

Reply:
I don't know why we have this, it seems to me that the req at 204 should be enough?

Reply:
As discussed, we probably should enforce reqs in tests (as opposed to suites), instead of duplicating this logic.